### PR TITLE
more occ fixes

### PR DESCRIPTION
**Pull request by mmattel**
These are more fixes to the occ command files
- Notification was in the wrong file
- Creating tables for Arguments and Options 
- Creating a unified look and feel for Arguments for Options
- Various optical fixes

### CODE REVIEW

File: modules/ROOT/pages/configuration/server/occ_app_command.adoc [hunk 1]
```diff
@@ -421,56 +420,56 @@ All of the available keys, along with default values for configValue, are listed
 
 [width="70%",cols=",",options="header",]
```
Comment (settermjd) [inline modules/ROOT/pages/configuration/server/occ_app_command.adoc]: Interesting. I wonder why GitHub renders some of the table italicised.

File: modules/ROOT/pages/configuration/server/occ_command.adoc [hunk 2]
```diff
@@ -975,17 +994,17 @@ files_external:backends [options]
 
 ==== Arguments
 
-[width="100%",cols="18%,70%",]
+[width="100%",cols="20%,70%",]
 |===
-| `type`    | Only show backends of a certain type. Possible values are `authentication` or `storage`.
-| `backend` | Only show information of a specific backend.
+| `type`    | Only show backends of a certain type. Possible values are `authentication` or `storage`
+| `backend` | Only show information of a specific backend
```
Comment (settermjd) [inline modules/ROOT/pages/configuration/server/occ_command.adoc]: Why are the full-stops being removed from the trailing sentences?

File: modules/ROOT/pages/configuration/server/occ_app_command.adoc (hunk 1 above)
Comment (mmattel) [reply modules/ROOT/pages/configuration/server/occ_app_command.adoc]: No clue, I double checked but I can´t find the reason.

File: modules/ROOT/pages/configuration/server/occ_command.adoc [hunk 3]
```diff
@@ -377,11 +382,13 @@ System config value maintenance set to boolean false
...
 Some configurations (e.g., the trusted domain setting) are an array of
-data. In order to set (and also get) the value of one key, you can
-specify multiple `config` names separated by spaces:
+data. The array starts counting with 0. In order to set (and also get) 
+the value of one key, you can specify multiple `config` names separated 
+by spaces:
 
 ....
-sudo -u www-data php occ config:system:get trusted_domains localhost owncloud.local sample.tld
+sudo -u www-data php occ config:system:get trusted_domains
```
[the remark this answers is not in the document]
Comment (mmattel) [reply modules/ROOT/pages/configuration/server/occ_command.adoc]: It is the get command, not the set command. It was misformatted before.
``config:system:get trusted_domains`` prints on the next line ``localhost owncloud.local sample.tld``

File: modules/ROOT/pages/configuration/server/occ_command.adoc [hunk 4]
```diff
@@ -391,7 +398,8 @@ be set:
 sudo -u www-data php occ config:system:set trusted_domains 2 --value=example.com
 System config value trusted_domains => 2 set to string example.com
 
-sudo -u www-data php occ config:system:get trusted_domains localhost owncloud.local example.com
+sudo -u www-data php occ config:system:get trusted_domains
```
[the remark this answers is not in the document]
Comment (mmattel) [reply modules/ROOT/pages/configuration/server/occ_command.adoc]: As above 😃

File: modules/ROOT/pages/configuration/server/occ_command.adoc [hunk 5]
```diff
@@ -600,9 +608,17 @@ first put your ownCloud server into
 single-user mode <maintenance_commands> to prevent any user
 activity until decryption is completed.
 
-- `-m` stands for the method. Method accepts the arguments `recovery` or `password`. If the _recovery_ method is chosen then the recovery password will be used to decrypt files. If the _password_ method is chosen then individual user passwords will be used to decrypt files.
+Arguments
 
-- `-c` accepts `yes` or `no`. This lets the command know whether to ask for permission to continue or not.
+[width="100%",cols="20%,70%",]
+|===
+| `-m=[METHOD]` | Accepts the arguments `recovery` or `password`. +
```
[the remark this answers is not in the document]
Comment (mmattel) [reply modules/ROOT/pages/configuration/server/occ_command.adoc]: I tried, but it looks not that good, but I can put it in a new line below...

File: modules/ROOT/pages/configuration/server/occ_command.adoc (hunk 2 above)
Comment (mmattel) [reply modules/ROOT/pages/configuration/server/occ_command.adoc]: uups... will fix

File: modules/ROOT/pages/configuration/server/occ_command.adoc (hunk 3 above)
[the remark this answers is not in the document]
Comment (settermjd) [reply modules/ROOT/pages/configuration/server/occ_command.adoc]: Ah, ok. FWIW, that's not clear from reading the text.

File: modules/ROOT/pages/configuration/server/occ_command.adoc (hunk 4 above)
[the remark this answers is not in the document]
Comment (settermjd) [reply modules/ROOT/pages/configuration/server/occ_command.adoc]: np.

File: modules/ROOT/pages/configuration/server/occ_command.adoc (hunk 5 above)
[the remark this answers is not in the document]
Comment (settermjd) [reply modules/ROOT/pages/configuration/server/occ_command.adoc]: At your discretion.

File: modules/ROOT/pages/configuration/server/occ_app_command.adoc (hunk 1 above)
Comment (settermjd) [reply modules/ROOT/pages/configuration/server/occ_app_command.adoc]: Well, that's what counts.

File: modules/ROOT/pages/configuration/server/occ_command.adoc (hunk 3 above)
[the remark this answers is not in the document]
Comment (mmattel) [reply modules/ROOT/pages/configuration/server/occ_command.adoc]: printing found items is one item per line - changed accordingly (fixed)

**Pull request merged**